includes/definitions/discovery/bdcom.yaml

```diff
@@ -1,13 +1,23 @@
-mib: BDCOM-PROCESS-MIB
+mib: NMS-CHASSIS
 modules:
     mempools:
         data:
             -
-                used: BDCOM-MEMORY-POOL-MIB::bdcomMemoryPoolUsed
-                free: BDCOM-MEMORY-POOL-MIB::bdcomMemoryPoolFree
+                percent_used: .1.3.6.1.4.1.3320.3.6.10.1.12.0
+                descr: 'Memory Usage'
+    os:
+        hardware: .1.3.6.1.4.1.3320.3.6.10.1.5.0
+        version: .1.3.6.1.4.1.3320.3.6.10.1.6.0
+        serial: .1.3.6.1.4.1.3320.3.6.10.1.4.0
+        features: .1.3.6.1.4.1.3320.3.6.10.1.3.0
     processors:
         data:
             -
-                oid: bdpmCPUTotal5min
-                num_oid: '.1.3.6.1.4.1.3320.9.109.1.1.1.1.5.{{ $index }}'
-                index: 0
+                oid: nmscardCPUUtilization
+                num_oid: '.1.3.6.1.4.1.3320.3.6.10.1.11.{{ $index }}'
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: nmscardTemperature
+                    num_oid: '.1.3.6.1.4.1.3320.3.6.10.1.13.{{ $index }}'
```

Comment:
Mempool, os and CPU works fine.  

Sensor temp, and voltage problem.  I have attached a mib file, the command is given a separate answer.

Expected "(" (_): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Should be ACCESS (1FastEthernet-100M): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Bad parse of OBJECT-TYPE: At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
nmscardCPUUtilization: Unknown Object Identifier (Sub-id not found: (top) -> nmscardCPUUtilization)  
sensors SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUs' '-Ih' '-m' 'NMS-CHASSIS' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/bdcom' 'udp:HOSTNAME:161' 'nmscardTemperature']
Exitcode: 1  
Expected "(" (_): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Should be ACCESS (1FastEthernet-100M): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Bad parse of OBJECT-TYPE: At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
nmscardTemperature: Unknown Object Identifier (Sub-id not found: (top) -> nmscardTemperature)  
  
Expected "(" (_): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Should be ACCESS (1FastEthernet-100M): At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
Bad parse of OBJECT-TYPE: At line 287 in /opt/librenms/mibs/bdcom/NMS-CHASSIS
nmscardTemperature: Unknown Object Identifier (Sub-id not found: (top) -> nmscardTemperature)

Reply:
net-snmp can't parse the mib.  You'll need to alter the MIB to make it work.

Reply:
"murrant30.11.2020
@hartred the mib is invalid. If you want it to work, you need to replace the mentioned _ with -"    can u show some example?

Reply:
Fix that. Edit yaml now and re-upload later